scripts/process_simple_articles.py

```
@@ -0,0 +1,91 @@
+                        'is_relevant_article',
+                        article=article
+                    )
+                    if is_relevant:
+                        # Mark as processed
+                        storage.update_article_status(
+                            article.get('id') or article.get('guid'),
+                            'processed',
+                            {'score': 50, 'processed_at': date.today().isoformat()}
+                        )
+                        relevant_articles.append(article)
+                except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)